app/serializers/proposal_search_serializer.rb

```diff
@@ -1,9 +1,24 @@
 class ProposalSearchSerializer < ActiveModel::Serializer
   attributes(
     :total,
+    :size,
+    :from,
+    :current_page,
     :proposals
   )
 
+  def size
+    object.size
```

Comment:
we we need to define these if they are just delegating to the serialized object?